### PR TITLE
Process newsletter msgs

### PR DESCRIPTION
**Pull request by oscarguindzberg**
Fixes https://github.com/WhiskeySockets/Baileys/issues/628

### CODE REVIEW

File: src/Utils/decode-wa-message.ts [hunk 1]
```diff
@@ -78,6 +78,10 @@ export function decodeMessageNode(
 
 		chatId = from
 		author = participant
+	} else if(isJidNewsletter(from)) {
+		msgType = 'newsletter'
+		chatId = from
+		author = ''
```
Comment (azudindaem) [inline src/Utils/decode-wa-message.ts]: author = from

**Review by azudindaem — CHANGES_REQUESTED** [review]

File: src/Utils/decode-wa-message.ts (hunk 1 above)
Comment (oscarguindzberg) [reply src/Utils/decode-wa-message.ts]: done

**Review by ainitech — APPROVED** [review]
Ok from here! Tested!

**Comment by CSFelix** [comment]
The PR worked for me!!

**Review by CSFelix — APPROVED** [review]
LGTM!!

**Comment by PurpShell** [comment]
#919

**Pull request closed without merging**